applications/tm-forum-api/values.yaml

```diff
@@ -0,0 +1,75 @@
+tm-forum-api:
+  ## configuration to be used by every api-deployment if nothing specific is provided.
+  defaultConfig:
+  
+    # -- configuration to be used for the image of the container
+    image:
+      # -- repository to get the container from
+      repository: quay.io/wi_stefan
+      # -- tag to be used, most of the time the apis will use the same version
+      tag: 0.5.0
+      # -- pull policy to be used
+      pullPolicy: IfNotPresent
+
+    # --  ngsi-ld broker connection information
+    ngsiLd:
+      # -- timeout for requests ot the broker
+      readTimeout: 30s
+      # -- base path for the ngsi-ld api
+      path: ngsi-ld/v1
+      # -- address of the broker
+      url: http://orion:1026
+
+  # -- list of apis to be deployed
+  # -- every api needs to define a name and image. Basepath is required when routes or ingress will be enabled. Beside that, they can overwrite the default-config by using the same keys.
+  # -- be aware: when you change the image repositrory or the tag for an api, you have to provide both values for the changes to take effect
+  apis:
+    - name: party-catalog
+      image: tmforum-party-catalog
+      basePath: /tmf-api/party/v4
+      specUrl: https://tmf-open-api-table-documents.s3.eu-west-1.amazonaws.com/OpenApiTable/4.0.0/swagger/TMF632-Party-v4.0.0.swagger.json
```

Comment:
spec url is only required if you use the "doc" tool from where you copied the chart;)